library/src/main/java/com/novoda/downloadmanager/MigrationServiceBinder.java

```diff
@@ -8,31 +8,42 @@
 
 public class MigrationServiceBinder {
 
-    private final Context context;
-    private final Migrator.Callback migrationCallback;
+    private final Context applicationContext;
```

Comment:
❓ QUESTION ❓ 

Anyone know if there is an annotation to enforce `ApplicationContext`?

Reply:
The only defensive strategy is to eagerly get the application context from the provided context. If the provided context is already an application context it just returns itself

Reply:
No as far as I know, as is a `Context` also